cfme/containers/provider/__init__.py

```diff
@@ -91,15 +91,26 @@ class hawkular(Tab, BeforeFillMixin):  # NOQA
             Version.lowest(): Input('hawkular_api_port'),
             '5.9': Input('metrics_api_port')
         })
+
         validate = Button('Validate')
 
     @View.nested
     class alerts(Tab, BeforeFillMixin):  # NOQA
         TAB_NAME = 'Alerts'
-        sec_protocol = BootstrapSelect(id='prometheus_alerts_security_protocol')
-        trusted_ca_certificates = TextInput('prometheus_alerts_tls_ca_certs')
-        hostname = Input('prometheus_alerts_hostname')
-        api_port = Input('prometheus_alerts_api_port')
+
```

Comment:
I don't think these version picks are necessary, as there is only one value anyway.

If we wanted to model that this tab isn't in the view on 59z, the entire view would have to be extended and version picked, which I do not think is necessary in this case.

I think you can leave this class as-is.

Reply:
I will revert this code.